conditions/conditions.go

```diff
@@ -69,12 +69,12 @@ func NewCondition(cl client.Client, condType apiv1.ConditionType) (Condition, er
 
 // Get implements conditions.Get
 func (c *condition) Get(ctx context.Context) (*metav1.Condition, error) {
-	operatorCond := &apiv1.OperatorCondition{}
+	operatorCond := &apiv2.OperatorCondition{}
 	err := c.client.Get(ctx, c.namespacedName, operatorCond)
 	if err != nil {
 		return nil, ErrNoOperatorCondition
```

Comment:
Please consider returning the original error, or at least wrap it.

Reply:
I support this change, but recommend doing so in a separate commit.

Reply:
See https://github.com/operator-framework/operator-lib/pull/65